### PR TITLE
boilerplate create-react-app hello world

### PR DESCRIPTION
**Pull request by terrilldent**
This is the generic hello world setup for a "create-react-app"

To run locally you may need to run these commands:

```
npm install -g npx
yarn install
yarn start
```

### CODE REVIEW

File: exemplar/web/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "exemplar",
+  "version": "0.1.0",
```
Comment (swankjesse) [inline exemplar/web/package.json]: for me to follow up – are all of these required? Can we omit the version etc.?

File: exemplar/web/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (swankjesse) [inline exemplar/web/public/manifest.json]: neat

**Comment by mmihic** [comment]
High level - do we want this in exemplar, or in another example app? I'm a little wary of exemplar becoming a grab bag of random examples of stuff making it confusing and not so useful as an example app. Think we should consider having a family of example apps each of which is targeted at specific subsets of misk (transactional RPC style app, webapps, background task app, app using various cloud services, etc).

File: exemplar/web/package.json (hunk 1 above)
Comment (eeSeeGee) [reply exemplar/web/package.json]: You can specifiy a minimum version but that's just asking for trouble.

**Comment by swankjesse** [comment]
https://github.com/square/misk/pull/146

**Pull request merged**